appinventor/appengine/war/WEB-INF/appengine-web.xml

```diff
@@ -93,8 +93,9 @@
     <property name="localauth.mailserver.password" value="changeme" />
 
     <!-- host[:port] to use for connecting to the build server -->
-    <property name="build.server.host" value="localhost:9990" />
+    <property name="build.server.host" value="visual-access.csail.mit.edu:18898" />
```

Comment:
This looks extraneous, please remove changes from this file.

Reply:
Yes, I've removed it and pushed now.